src/status_im/ui/screens/db.cljs

```diff
@@ -223,6 +227,9 @@
                  :node/status
                  :node/restart?
                  :node/address
+                 :inbox/fetching?
```

Comment:
why is inbox/fetching? back ?

Reply:
probably a merge artifact, will clean it up